src/lib/common/JsonHelper.cpp

```diff
@@ -161,9 +162,6 @@ std::string objectToJson(std::map<std::string, std::string>& list)
 */
 JsonHelper::JsonHelper(): empty(true)
 {
-  /* Set format  for floats (it doesn't affect integers) */
```

Comment:
This has been moved to the particular function dealing with float printing, i.e. `addFloat()`.